jenkins-builds/leone-RHEL6.9EUS-17.06

```diff
@@ -0,0 +1,14 @@
+ Boost-1.61.0-foss-2016b-Python-2.7.12.eb
+ ddt-18.0.1.eb
```

Comment:
one version of ddt should be enough?

Reply:
would be the right time to update the recipe (and not use /apps/common)

Reply:
👍

Reply:
https://github.com/eth-cscs/production/pull/832